src/components/content/heroes/aurora/weather-easter-egg/weather-api.ts

```diff
@@ -0,0 +1,96 @@
+import axios from 'axios';
+import { WeatherType } from './weather-types';
+
+const API_KEY = process.env.GATSBY_WEATHER_API_KEY;
+const BASE_URL = 'https://api.weatherapi.com/v1';
+
+interface getSunTimeProps {
+  sunriseTime: number;
+  sunsetTime: number;
+}
+
+export const getWeather = async (): Promise<WeatherType> => {
```

Comment:
It's okay to remove the type. Typescript will determine it correctly.
Remove and check your IDE which will show the correct type;

<img width="753" alt="image" src="https://github.com/satellytes/satellytes.com/assets/1701755/585b699c-25de-4fbe-b660-3e0ad3e8d48b">

That's something many people do wrong because they think they have to declare it while they don't. Other folks just prefer to be explicit but I don't see any benefit in "over declaring" because pure functions are easily readable including the involved type.